visual-challenge/src/test/java/applitools/hackathon/TestsInit.java

```diff
@@ -0,0 +1,29 @@
+package applitools.hackathon;
+
+/**
+ * Created by Roman Iovlev on 10.11.2018
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+
+import applitools.hackathon.pages.ApplitoolsSite;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+
+import static com.epam.jdi.light.driver.WebDriverUtils.killAllSeleniumDrivers;
+import static com.epam.jdi.light.elements.init.PageFactory.initSite;
+import static com.epam.jdi.tools.PropertyReader.readProperties;
+
+public class TestsInit {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'TestsInit' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4513258)